tests/php/bootstrap.php

```diff
@@ -11,6 +11,9 @@
 require_once $amp_plugin_root . '/vendor/xwp/wp-dev-lib/sample-config/phpunit-plugin-bootstrap.php';
 
 /**
- * Load WP CLI.
+ * Load WP CLI. Its test bootstrap file can't be required as it will load
+ * duplicate class names which are already in use.
  */
-require_once $amp_plugin_root . '/vendor/wp-cli/wp-cli/tests/bootstrap.php';
+define( 'WP_CLI_ROOT', $amp_plugin_root . '/vendor/wp-cli/wp-cli' );
```

Comment:
I noticed a lot of warnings were occuring during the PHPUnit bootstrap process. It turns out WP CLI also has [implemented ](https://github.com/wp-cli/wp-cli/blob/d394b395995e70932df5a58baab18f2942afbc28/tests/bootstrap.php#L10-L12) PHPUnit 6+ compatibility, which Core already provides. The fix then would be to extract the needed parts from the WP CLI bootstrap file and into ours.

Reply:
Alternatively, to keep with the WP-CLI bootstrap file (in case that might add other stuff), we could try defining the `PHPUnit\Runner\Version` class at this point... Not sure that will work as expected, though.

Reply:
I didn't look into _when_ `PHPUnit\Runner\Version` is loaded, but it does certainly seem to be before the PHPUnit bootstrap file is loaded.

Reply:
Oh, nevermind, the logic is the wrong way around anyway. Let's copy over what we need then.